app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: :User, foreign_key: :user_id
```

Comment:
- [ ] Foreign key isn't required here, as it should be already established in the `user` model. Simply, _Post_ belongs to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.

Reply:
Done